.pulumi.version

```diff
@@ -0,0 +1 @@
+v3.156.0
```

Comment:
What is going to update this? Shall we fix up this job to keep it in sync?

https://github.com/pulumi/pulumi-terraform-bridge/blob/fa8e83f1f6e965d144bd9fcb634a379de79e1535/.github/workflows/weekly-pulumi-update.yml#L1

Reply:
Separate PR perhaps. We really need this to unblock work.